### PR TITLE
Set renovate bot to update cypress daily and all other dependencies on weekends

### PR DESCRIPTION
**Pull request by kevinold**

### CODE REVIEW

File: renovate.json [hunk 1]
```diff
@@ -17,7 +17,12 @@
     "enabled": true
   },
   "masterIssue": true,
+  "schedule": ["every weekend"],
   "packageRules": [
+    {
+      "packageNames": ["cypress"],
+      "extends": ["every weekday"]
```
Comment (bahmutov) [inline renovate.json:24]: hmm, is this valid syntax? it is weird to extend a schedule. I would think something like

```
{
  "packageNames": ["cypress"],
  "schedule": "every weekday"
}
```

Comment (kevinold) [reply renovate.json:24]: Yes, per their docs
Screen Shot 2020-09-09 at 4 05 32 PM

Comment (kevinold) [reply renovate.json:24]: Good catch

Comment (kevinold) [reply renovate.json]: Fixed per https://docs.renovatebot.com/presets-schedule/#scheduleweekdays

**Review by amirrustam — APPROVED** [review]

**Pull request merged**